plugins/pastefromword/filter/default.js

```diff
@@ -95,7 +95,8 @@
 						}
 					};
 
-					if ( element.parent ) {
+					// If the parent is DocumentFragment it does not have any attributes. #16912
+					if ( element.parent && element.parent.type != CKEDITOR.NODE_DOCUMENT_FRAGMENT ) {
```

Comment:
Fix is good, but I'd go with duck-typing here, so that it gives us even better protection. So I suggest checking `element.parent.attributes`.